be/src/agent/task_worker_pool.cpp

```diff
@@ -626,6 +626,10 @@ void TaskWorkerPool::_alter_tablet(
             finish_tablet_infos.push_back(tablet_info);
         }
     }
+    
+    for (auto& tablet_info : finish_tablet_infos) {
+        LOG(INFO) << "finish tablet infos:" <<  apache::thrift::ThriftDebugString(tablet_info);
+    }
```

Comment:
It should be debug or vlog or there will be too much logs.

Reply:
ok